packages/foreman/nodejs-theforeman-vendor/nodejs-theforeman-vendor.spec

```diff
@@ -4,16 +4,14 @@
 %global npm_name @theforeman/vendor
 
 Name: %{?scl_prefix}nodejs-theforeman-vendor
-Version: 1.7.0
-Release: 3%{?dist}
+Version: 3.3.2
+Release: 1%{?dist}
 Summary: foreman supported 3rd-party node_modules
 License: MIT
 Group: Development/Libraries
 URL: https://github.com/theforeman/foreman-js#readme
 Source0: https://registry.npmjs.org/%{npm_name}/-/%{npm_name}-%{version}.tgz
-%if 0%{?scl:1}
-BuildRequires: %{?scl_prefix_nodejs}npm
-%else
+%if 0%{?!scl:1}
```

Comment:
Meh, something is wrong with my nom2rpm..

Reply:
I kind of assumed that was intended since non-bundled packages don't really call NPM. Or is it needed for macros?

Reply:
Well, I don't know. It wasn't intended, so I was going with whatever was there before

Reply:
I still backed out that change, will revisit later.